### DISCUSSION TITLE
Blitzjs with React 17

### DISCUSSION THREAD

**Discussion by lcswillems** [blitz-js/blitz#1863] [Q&A] [answered]
### What do you want and why?

Blitz.js currently uses an experimental version of React. I assume that it is used because Blitz.js needs some experimental features of React. What are them? Would Blitz.js work without them?

**Comment by merodiro** [comment, marked as answer]
> What are them?

Concurrent Mode. it's currently not available in the stable release and it's only available in the experimental channel

> Would Blitz.js work without them?

I believe you can turn off concurrent mode by changing `reactMode` but I'm not sure if you can use the stable release from react

**Comment by lcswillems** [comment]
Thank you for your answer! I saw it uses suspense too. I will investigate.

**Reply by jason-nivelo** [reply]
What were your findings?
